src/modules/components/card.js

```diff
@@ -0,0 +1,49 @@
+import commentPop from './commentpop';
+import InvolvementAPI from '../classes/involvementapi';
+
+export default function createCard(mealName, strMealImage, idMeal, likes) {
```

Comment:
**Changes Required**

- [ ] Thanks for implementing these functionalities. However, I suggest we should prefer the use of `ES6 Arrow functions`. Let us try and refactor these functions using the `ES6 Arrow Functions.`